content/aips/AAVE_ETHEREUM_V3.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Aave Ethereum V3
+author: BGD Labs (@bgdlabs)
+shortDescription: Listing WBTC, WETH, wstETH, USDC, DAI, LINK, AAVE and activating Ethereum V3 Pool
+discussions: https://governance.aave.com/t/bgd-aave-v3-ethereum-new-deployment-vs-aave-v2-upgrade/9990/16
+created: 2023-01-20
+---
+
+# Simple Summary
+
+This governance proposal executes the listing og WBTC, WETH, wstETH, USDC, DAI, LINK, AAVE on AAVE Ethereum V3.0.1 pool. It also creates an `ETH correlated` eMode category and activates the pool.
+
+# Motivation
+
+While AAVE V3 was deployed to various networks right after the release, the Ethereum pool was still running V2.
+After careful considerations, the community decided to deploy a fresh V3 instead of upgrading the V2 pool for increased compatibility between the V3 pools.
+The assets to be listed & respective configurations have been discussed in [the governance forum](https://governance.aave.com/t/arc-aave-v3-ethereum-deployment-assets-and-configurations/10238) and been decided in the following [snapshot](https://governance.aave.com/t/arc-aave-ethereum-v3-market-initial-onboarded-assets/11318/5).
+
+# Specification
+
+This proposal's payload does the following:
+
+1. activate the pool via `POOL_CONFIGURATOR.setPoolPause(false)`
+2. create the eMode category via `POOL_CONFIGURATOR.setEModeCategory(1, 90_00, 93_00, 10_100, address(0), 'ETH correlated')`
+3. pass admin permissions to the following contracts:
+   - TBA
+   - TBA
+   - TBA
+
+Each of these addresses follows the steward pattern used in [v3-listing stewards](https://github.com/bgd-labs/aave-v3-listing-stewards/blob/feat/v3-ethereum-tests/src/contracts/common/StewardBase.sol#L8). Each stewards will handle the listing of one specific asset and then renounce all it's roles. This process is needed as otherwise the proposal would not fit within a single transaction.
```

Comment:
```suggestion
Each of these addresses follows the Steward pattern used in [v3-listing stewards](https://github.com/bgd-labs/aave-v3-listing-stewards/blob/feat/v3-ethereum-tests/src/contracts/common/StewardBase.sol#L8). Each Steward will handle the listing of one specific asset and then renounce to the granted POOL_ADMIN role. This process is needed as otherwise, the proposal's gas limit would be really gas-expensive.
```